bedrock/firefox/templates/firefox/new/base.html

```diff
@@ -10,18 +10,33 @@
 
 {% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
 
-{# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
-{%- block page_title -%}
-  {% if LANG == 'en-CA' %}
+
+{% if switch('firefox_new_title_experiment') %}
+    {# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
+    {%- block page_title -%}
+    {% if LANG == 'en-CA' %}
+    The Fast and Free Web Browser 🦊 (CA)
```

Comment:
@pmac are there any encoding gotcha's about including emoji like this that you are aware of? I think this should be fine, but I may be unaware of edgecases.

Reply:
I'm actually not sure. It should be like any other unicode character in theory, so I'd say no, but I don't think we've done this exact thing before. So I'd say it "should" be okay, but we should keep an eye on it.

Reply:
Ok thanks. Yeah we're putting this behind a switch and only for a couple of locales. We can always flip it off if we notice anything weird. Thanks